src/methods/img-sizes.js

```diff
@@ -31,8 +31,16 @@ const imgSizes = img => {
  * @returns {object}
  *
  * @example
- * // TODO
- *
+ * // Can be a string as parameter
+ * const imgString = '200x50'
+ * ecomUtils.imgSizes(imgString)
+ * // return an object with width and height
+ * // => {width: 200, height: 50}
+ * // Can be an object as a parameter as well
```

Comment:
I think is better to create a new example block here

```suggestion
 *
 * @example
 * // Using image object as parameter
```